tests/unit/test_cube.py

```diff
@@ -340,7 +340,8 @@ def it_can_inflate_itself(self, request):
                         },
                         {"other": "dim"},
                     ]
-                }
+                },
+                "cube_idx": 1,
```

Comment:
Who needs to provide this part of the JSON? I it's us (from within cube), then it shouldn't live in the fixture.